new_files/admin/includes/extra/functions/composer_autoload.php

```diff
@@ -2,12 +2,17 @@
 
 declare(strict_types=1);
 
-if (file_exists(DIR_FS_DOCUMENT_ROOT . '/vendor-no-composer/autoload.php')) {
-    require_once DIR_FS_DOCUMENT_ROOT . '/vendor-no-composer/autoload.php';
+$autoloader_deprecated = DIR_FS_DOCUMENT_ROOT . '/vendor-no-composer/autoload.php';
+$autoloader = DIR_FS_DOCUMENT_ROOT . '/vendor-mmlc/autoload.php';
```

Comment:
Thx. :-)
1. Please use camelCase
2. This vars are in global scope so it is better to prefix them with "rth".